### PR TITLE
Allow opening tickets from UI

### PR DESCRIPTION
**Pull request by rowanseymour**
Captura de Pantalla 2022-07-18 a la(s) 10 45 31

### CODE REVIEW

File: temba/contacts/models.py [hunk 1]
```diff
@@ -660,6 +660,21 @@ def resolve(cls, channel, urn):
         contact_urn = ContactURN.objects.get(id=response["urn"]["id"])
         return contact, contact_urn
 
+    @classmethod
+    def from_urn(cls, org, urn_as_string, country=None):
```
Comment (rowanseymour) [inline temba/contacts/models.py:664]: sorry unrelated change - just was annoying me that this method was sitting in amongst all the methods for modifying contacts so moved it up here

**Review by norkans7 — APPROVED** [review]
